### PR TITLE
chore(samples): remove region tags.

### PR DESCRIPTION
**Pull request by kchertenko**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-retail/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by parthea** [comment]
@chenlei1216 Please could you confirm that you'd like to remove region tags?

**Comment by chenlei1216** [comment]
> @chenlei1216 Please could you confirm that you'd like to remove region tags?

Thanks for checking. I don't think they're used by our docs so I think it should be fine to remove.

**Review by chenlei1216 — APPROVED** [review]

**Comment by chenlei1216** [comment]
> > @chenlei1216 Please could you confirm that you'd like to remove region tags?
> 
> Thanks for checking. I don't think they're used by our docs so I think it should be fine to remove.

But I don't know what's the purpose of this PR. I assume the region tags are automatically generated. So they could be added back automatically later?

File: samples/generated_samples/retail_v2_generated_catalog_service_add_catalog_attribute_async.py [hunk 1]
```diff
@@ -23,7 +23,6 @@
 #   python3 -m pip install google-cloud-retail
 
 
-# [START retail_v2_generated_CatalogService_AddCatalogAttribute_async]
```
Comment (chenlei1216) [inline samples/generated_samples/retail_v2_generated_catalog_service_add_catalog_attribute_async.py:26]: Should we just skip the generated samples?

Comment (parthea) [reply samples/generated_samples/retail_v2_generated_catalog_service_add_catalog_attribute_async.py:26]: @kchertenko Please could you revert the changes to the autogenerated samples?

Comment (kchertenko) [reply samples/generated_samples/retail_v2_generated_catalog_service_add_catalog_attribute_async.py]: done

Comment (chenlei1216) [reply samples/generated_samples/retail_v2_generated_catalog_service_add_catalog_attribute_async.py]: SGTM now. Thanks

**Comment by chenlei1216** [comment]
> > > @chenlei1216 Please could you confirm that you'd like to remove region tags?
> > 
> > 
> > Thanks for checking. I don't think they're used by our docs so I think it should be fine to remove.
> 
> But I don't know what's the purpose of this PR. I assume the region tags are automatically generated. So they could be added back automatically later?

Discussed offline, that we're not gonna use the interactive tutorial samples in cloud doc, so SGTM to remove the tags.

**Review by parthea — APPROVED** [review]

**Pull request merged**